Hunting/Assets/Scripts/GameManager/SunController.cs

```diff
@@ -10,34 +10,95 @@ public class SunController : MonoBehaviour
     private float dayTime = 60;
 
     [SerializeField]
-    private float dimFactor = 3;
+    private float currentTimeOfDay = 0.5f;
+
+    private float secondsInFullDay = 120f;
+    private float timeMultiplier = 0.5f;
+
+    private MeshRenderer cloud;
     
     # endregion
 
     # region Fields
 
-    private float moveAngle = 0;
 
-    private new Light light = null;
+    private Light sun;
+    private float sunInitialIntensity;
+
+    private Material high = null;
+
+    private float r = 1f;
+    private float g = 1f;
+    private float b = 1f;
+
+
+    public System.Collections.Generic.List<Color> colors = new System.Collections.Generic.List<Color>();
 
     # endregion
 
     # region MonoBehaviour Methods
 
     void Start()
     {
-        if (dayTime != 0)
-            moveAngle = 60/dayTime;
         
-        light = GetComponent<Light>();
-        light.intensity = 1;
+        sun = GetComponent<Light>();
+
+        sunInitialIntensity = sun.intensity;
+
+        high = cloud.materials[0];
     }
 
     // Update is called once per frame
     void Update()
     {
-        transform.Rotate(Vector3.up * Time.deltaTime * moveAngle);
-        light.color += (Color.red/(dayTime * dimFactor)) * Time.deltaTime;
+
+        UpdateSun();
+        
+        currentTimeOfDay += (Time.deltaTime / secondsInFullDay) * timeMultiplier;
+ 
+        if (currentTimeOfDay >= 1) 
+	{

```

Comment:
A bit of nitpicking here - make the braces align if you can.

Reply:
![Capture](https://user-images.githubusercontent.com/28922936/75865949-46ae4c80-5e3f-11ea-9150-a967d5de31f8.PNG)

looks good on my end